openslides_backend/models/checker.py

```diff
@@ -46,6 +46,14 @@
     }
 )
 
+external_motion_fields = [
```

Comment:
remove this, see above

Reply:
But this is the main point of the PR: The error occured because the `check_database` presenter and `meeting.clone` needed the same functionality, but implemented them both manually. I fixed in an earlier PR the field list of the presenter, but of course did not think about other places which might hold the same list. By having the field list in one central place, such errors can no longer occur.